clusterloader2/pkg/prometheus/prometheus.go

```diff
@@ -46,9 +47,33 @@ const (
 	checkPrometheusReadyTimeout  = 15 * time.Minute
 	numK8sClients                = 1
 	nodeExporterPod              = "$GOPATH/src/k8s.io/perf-tests/clusterloader2/pkg/prometheus/manifests/exporters/node-exporter.yaml"
-	nodeExporterTemplateName     = "PROMETHEUS_SCRAPE_NODE_EXPORTER"
 )
 
+var (
```

Comment:
Given we're cleaning this up, I'm not a fan of global variables.

Instead, can we have a type with options and make those part of some config (maybe even cluster config) and initialize them with flags more explicitly.
This is one example of doing that:
https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/apiserver/pkg/server/options/etcd.go#L112

Reply:
Done.